docs/integrations/azure/locations.md

```diff
@@ -52,13 +76,19 @@ integrations:
 
 The configuration is a structure with these elements:
 
-- `host`: The DevOps host; only `dev.azure.com` is supported.
-- `token` (optional): A personal access token as expected by Azure DevOps.
-- `credential`: (optional): A service principal or managed identity
+- `credentials`: (optional): A service principal or managed identity
+
+The `credentials` element is a structure with these elements:
+
+- `organizations`: (optional): A list of organizations for which this credential should be used. Only supported for Azure DevOps (cloud). If not specified the credential will be used for all organizations for which no other credential is configured.
+- `clientId`: The client ID of the service principal or managed identity (required for service principal and managed identities)
+- `clientSecret`: The client secret of the service principal (required for service principal)
+- `tenantId`: The tenant ID of the service principal (required for service principal)
+- `token`: The personal access token (required for personal access token)
```

Comment:
If this is for a new `credentials` configuration array, is it worth renaming `token` to `personalAccessToken`.  `token` could be confused with `access token` used by the AAD auth methods?

Reply:
True, `token` is ambiguous, so I renamed it to `personalAccessToken`. It does seem redundant on the `PersonalAccessTokenCredential` type, but this type is mainly used as part of the `AzureDevOpsCredential` union type and in those cases `credential.personalAccessToken` is much more self-explanatory than `credential.token`

```typescript
export type PersonalAccessTokenCredential = AzureCredentialBase & {
  kind: 'PersonalAccessToken';
  personalAccessToken: string;
};
```